readme.md

```diff
@@ -1,31 +1,63 @@
-# spn
+# Starport Network
 
-**spn** is a blockchain application built using Cosmos SDK and Tendermint and generated with [Starport](https://github.com/tendermint/starport).
+Starport Network (SPN) is a network for launching [Cosmos](https://cosmos.network) blockchains by matching coordinators with validators. This repository contains an SPN node built with Cosmos SDK and generated with [Starport](https://github.com/tendermint/starport).
 
-## Get started
+## Using SPN
 
+To use SPN you will need [Starport](https://github.com/tendermint/starport) installed. SPN is being actively developed, please, build Starport from source and use `develop` branch.
+
+### Initiating a blockchain launch
+
+To initiate a blockchain launch run the following command:
+
+```
+starport network chain create --chain [chainID] --source [sourceURL]
+```
+
+`chainID` is a string that uniquely identifies your blockchain on SPN. `sourceURL` is a URL that can be used to clone the repository containing a Cosmos SDK blockchain node (for example, `https://github.com/tendermint/spn`). By running the `create` command you act as a "coordinator" and initiate the launch of a blockchain.
+
+By default a coordinator does not propose themselves as a validator. To do so, run `join` command and your proposal will be automatically approved.
```

Comment:
The user may not know the notion of proposals and approvals at this point.
We may need a short introduction at the beginning of the readme.

- We record information of a blockchain to launch as a "chain"
- Users can send proposals to the chain. Proposals represent an additional piece for the launch of the blockchain (validator, account, genesis modification soon)
- Can be approved or rejected by the coordinator

We may also need to add some context on why we need this. Though we could put a link in the future (I think there will be a place in the starport documentation that contains a complete introduction to Starport Network)